itests/eth_transactions_test.go

```diff
@@ -83,6 +83,11 @@ func TestValueTransferValidSignature(t *testing.T) {
 
 	// Success.
 	require.EqualValues(t, ethtypes.EthUint64(0x1), receipt.Status)
+
+	ethTx, err := client.EthGetTransactionByHash(ctx, &hash)
+	require.Nil(t, err)
+	require.EqualValues(t, ethAddr, ethTx.From)
```

Comment:
Let's also assert against all `tx` fields? That way if something else fails to populate/breaks, we'll catch it too.

Reply:
Won't be able to do it today, but happy to land this later this week 🙂